tests/acceptance/features/apiProvisioningLDAP/users.feature

```diff
@@ -129,6 +129,101 @@ Feature: Manage users using the Provisioning API
     # | 1               | 102             | 200              |
     # | 2               | 400             | 400              |
 
+  @issue-core-33186
+  Scenario Outline: admin tries to modify quota of user for which an LDAP attribute is specified
+    Given using OCS API version "<ocs-api-version>"
+    #to set Quota we can just misuse any LDAP text field
+    And LDAP config "LDAPTestId" has key "ldapQuotaAttribute" set to "employeeNumber"
+    When the administrator sets the ldap attribute "employeeNumber" of the entry "uid=user1,ou=TestUsers" to "10 MB"
+    And the LDAP users are resynced
+    And the administrator changes the quota of user "user1" to "13MB" using the provisioning API
+    Then the OCS status code should be "<ocs-status-code>"
+    And the HTTP status code should be "<http-status-code>"
+    And the quota definition of user "user1" should be "13 MB"
+    #And the quota definition of user "user1" should be "10 MB"
+    When the LDAP users are resynced
+    Then the quota definition of user "user1" should be "10 MB"
+    Examples:
+      | ocs-api-version | ocs-status-code | http-status-code |
+      | 1               | 100             | 200              |
+      | 2               | 200             | 200              |
+    # | 1               | 102             | 200              |
+    # | 2               | 400             | 400              |
+
+  Scenario Outline: admin sets quota of user for which no LDAP quota attribute is specified
+    Given using OCS API version "<ocs-api-version>"
+    #to set Quota we can just misuse any LDAP text field
+    And LDAP config "LDAPTestId" has key "ldapQuotaAttribute" set to "employeeNumber"
+    And the LDAP users have been resynced
+    When the administrator changes the quota of user "user1" to "13MB" using the provisioning API
+    Then the OCS status code should be "<ocs-status-code>"
+    And the HTTP status code should be "<http-status-code>"
+    And the quota definition of user "user1" should be "13 MB"
+    When the LDAP users are resynced
+    Then the quota definition of user "user1" should be "13 MB"
+    Examples:
+      | ocs-api-version | ocs-status-code | http-status-code |
+      | 1               | 100             | 200              |
+      | 2               | 200             | 200              |
+
+  @issue-core-33186
+  Scenario Outline: admin sets quota of user for which no LDAP quota attribute is specified but the default quota setting is set
+    Given using OCS API version "<ocs-api-version>"
+    #to set Quota we can just misuse any LDAP text field
+    And LDAP config "LDAPTestId" has key "ldapQuotaAttribute" set to "employeeNumber"
+    And LDAP config "LDAPTestId" has key "ldapQuotaDefault" set to "10MB"
```

Comment:
In the scenario outline you say "for which **no** LDAP quota attribute is specified" but then you specify a LDAP quota attribute in this line? Do I misunderstand?

Reply:
Ah, it refers to the **default** quota attribute in LDAP, right?

Reply:
yes